group.lisp

```diff
@@ -542,6 +542,19 @@ The windows will be moved to group \"^B^2*~a^n\"
             (message "Canceled"))
         (message "There's only one group left"))))
 
+(defcommand gkill-other () ()
+  "Kill other groups. All windows in other groups are migrated
+to the current group."
+  (let* ((current-group (current-group))
+         (groups (remove-if (lambda (g) (eq g current-group))
```

Comment:
`(remove current-group (screen-groups (current-screen))` would be better

Reply:
:+1: